data/json/recipes/recipe_others.json

```diff
@@ -5050,7 +5050,39 @@
       [ "silver_small", 50 ]
      ]
     ]
-},{
+},
+{
+  "type" : "recipe",
+  "result": "mirror",
```

Comment:
Oops, something I neglected to mention.  There's already a recipe for "mirror", so this conflicts.  We add an "id_suffix" line to fix this; I'll go with "from_steel".